app/javascript/dashboard/components/ChatList.vue

```diff
@@ -685,6 +686,21 @@ export default {
         this.showAlert(this.$t('BULK_ACTION.LABELS.ASSIGN_FAILED'));
       }
     },
+    async onAssignTeamsForBulk(team) {
+      try {
+        await this.$store.dispatch('bulkActions/process', {
+          type: 'Conversation',
+          ids: this.selectedConversations,
+          fields: {
+            team_id: team.id,
+          },
+        });
+        this.selectedConversations = [];
+        this.showAlert(this.$t('BULK_ACTION.TEAMS.ASSIGN_SUCCESFUL'));
+      } catch (err) {
+        this.showAlert(this.$t('BULK_ACTION.TEAMS.ASSIGN_FAILED'));
```

Comment:
Check any messages that exist in the error payload otherways show the default message.

Reply:
![Screenshot2022-11-18 at 11 55 31](https://user-images.githubusercontent.com/15716057/202635450-b5b24dc8-5ba3-4eb0-b04c-a8fc9ea030eb.png)

@muhsin-k We're not sending any error messages right now for this api.